drivers/SmartThings/zigbee-switch/src/zigbee-dual-metering-switch/init.lua

```diff
@@ -25,31 +36,59 @@ local function can_handle_zigbee_dual_metering_switch(opts, driver, device, ...)
   return false
 end
 
-local function endpoint_to_component(device, endpoint)
-  if endpoint == 2 then
-    return "switch1"
-  else
-    return "main"
+local function do_refresh(self, device)
+  device:send(OnOff.attributes.OnOff:read(device))
+  device:send(SimpleMetering.attributes.Divisor:read(device))
+  device:send(SimpleMetering.attributes.Multiplier:read(device))
+  device:send(ElectricalMeasurement.attributes.ACPowerDivisor:read(device))
+  device:send(ElectricalMeasurement.attributes.ACPowerMultiplier:read(device))
```

Comment:
- Reading divisor and multiplier might be done once after inclusion (but please double check if defaults do it already at some point)
- DTH reads from ELECTRICAL_MEASUREMENT_CLUSTER, attribute 0x050B
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/78747fa5a3b42a0c801f1095f8f7521da968ba42/devicetypes/smartthings/zigbee-multi-switch-power.src/zigbee-multi-switch-power.groovy#L128